force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,22 +48,25 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer difference = a - b;
+        return difference; //PASS
```

Comment:
for a lot of these, you don't need that temp variable and can return directly. 

return a - b;